### DISCUSSION TITLE
UERANSIM gNB Multiple PLMN support

### DISCUSSION THREAD

**Discussion by JoaoFerreira5** [aligungr/UERANSIM#610] [General]
Hi,

Does UERANSIM support multiple PLMN selection in the same gNB? 
If not, is there a lot to change to accomplish this feature?

Best regards,
JoaoFerreira5

**Comment by jamjad333** [comment]
@JoaoFerreira5 did you figure out something?

**Reply by JoaoFerreira5** [reply]
@jamjad333 UERANSIM does not support multiple PLMN for the same gNodeB and I didn´t implement it. Instead, I simply used multiple VMs with different PLMNs since my goal was to test the CN, not the RAN.
